spec/cypress/support/step_definitions/notifications.js

```diff
@@ -0,0 +1,29 @@
+import { Given, When } from "cypress-cucumber-preprocessor/steps";
+
+Given(
+  "An email sign in notification should be sent for email {string}",
+  (email) => {
+    cy.verifySignInEmailSentForEmail(email);
+  }
+);
+
+Given(
+  "An Admin account created email will be sent to the email {string}",
+  (email) => {
+    cy.verifyAdminAccountCreatedEmailSentForEmail(email);
+  }
+);
+
+Given(
+  "Email should be sent to Primary Email Contact of the School belonging to {string}",
+  (email) => {
+    cy.verifyPrimaryContactEmailSentForEmail(email);
+  }
+);
```

Comment:
"will be sent", "should be sent", ...

I would standardise to say "xxx email was/is/should be/will be sent to {string}". And stick with the sentence type. Out of them, I would go for 'should be sent'.

Reply:
Yep, we've been using "should" a lot already

Reply:
Changed to should